docs/lib/datastore/menu.json

```diff
@@ -4,6 +4,7 @@
     "getting-started",
     "data-access",
     "relational",
+    "setup-auth-rules",
```

Comment:
I would put this after "sync" because the auth rules aren't really useful until the customer has setup a cloud backend.